docs/pbc/all/business-intelligence/202410.0/amazon-quicksight-third-party-integration/best-practices-analyzing-shop-data.md

```diff
@@ -0,0 +1,158 @@
+---
+title: "Best practices: Analyzing shop data"
+description: Learn how to analyze and present shop data.
+last_updated: Oct 24, 2024
+template: back-office-user-guide-template
+related:
+---
+
+Amazon Quicksight is a powerful business intelligence tool. It's highly customizable and has a lot of tools that can't be covered in one document. This document describes a typical use case and how you can start working with analytics, but doesn't cover all the editing options.
+
+For more insights into creating analyses and dashboards, see [Author Workshop](https://catalog.workshops.aws/quicksight/en-US/author-workshop).
+
+## Prerequisites
+
+[Set up Amazon QuickSight](/docs/pbc/all/business-intelligence/202410.0/amazon-quicksight-third-party-integration/set-up-amazon-quicksight.html)
+
+
+## Optional: Back up the default analysis
+
+The Analytics feature is shipped with a default analysis that can be useful while you're learning how to create your own. If you need to play around with the existing analysis, we recommend duplicating the default one as follows:
+
+1. In the Back Office, go to **Analytics**.
+2. On the **Analytics** page, click **Analyses**.
+3. Click on **MASTER Analysis**.
+4. On the page of the analysis, click **File**>**Save as Analysis**.
+5. In the **Save a copy** window, enter an **Analysis name**.
+6. Click **SAVE**.
+  This duplicates the analysis and opens it. Now you can safely play around with this duplicate or the original analysis.  
+
+
+
+## Refresh data
```

Comment:
here we need to ultimately mention that it makes sense to set up a scheduler first. No one will go and do it manually. But this process we already described on a separate page. Not sure why we would need this section again…